tests/helper/helper_kubectl.go

```diff
@@ -212,8 +212,10 @@ func (kubectl KubectlRunner) WaitAndCheckForTerminatingState(resourceType, names
 }
 
 func (kubectl KubectlRunner) VerifyResourceDeleted(ri ResourceInfo) {
```

Comment:
We will need to add one for OcRunner as well, no?
https://github.com/openshift/odo/blob/main/tests/helper/helper_oc.go#L586

Reply:
Yes, that would be more consistent (I didn't see any error on OpenShift for this test)

Reply:
I've also changed on OcRunner. Thanks